DentOS_Framework/DentOsTestbed/src/dent_os_testbed/constants.py

```diff
@@ -35,6 +35,7 @@
 # consistent
 
 PYTEST_SUITES = {
+    "suite_bridging":"Functional bridging tests",
```

Comment:
Let's use the following naming convention:
suite_functional_[FEATURE]

So in your case, this will be: suite_functional_bridging